src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -373,6 +421,7 @@ static bool equalMetadataValues(BSONObj& md1, BSONObj& md2)
 */
 static bool equalMetadataVectors(BSONObj& mdV1, BSONObj& mdV2)
```

Comment:
Yes, I agree.
Especially right before a new release ...

However, we have plenty of 'utility functions' like this one working on types.
If they all were methods of their classes, the reusage of these functions would be so much easier.

Let's make an issue.